CHANGELOG.md

```diff
@@ -28,6 +28,28 @@
 
   ([PR #N](https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/N))
 
+🆕 New features:
+
+- New component: Digital Marketplace Alert component
+
+  *Requirement before using this component:*
+  - To use this the application must be using `govuk-frontend` template
+  - To use this the application must be using `govuk-frontend` phase banner as phase tag
+    is no longer part of the header and sits underneath and outside the header
+
+  *Installing component:*
+  1. Check config.py has this line for jinja to find the components
+      ```
+                  os.path.join(repo_root, 'node_modules', 'digitalmarketplace-govuk-frontend')
+      ```
+
+  2. Import the component in `_base_page.html`
+     `{% from "digitalmarketplace/components/alert/macro.njk" import dmAlert %}`
+
+  3. Use the component `{{ dmAlert({}) }}`
```

Comment:
Maybe place a `...` or something in the params of this call and add "See examples in review app or refer to the readme."